packages/meditrak-server/src/routes/userEntityPermissions/assertUserEntityPermissionPermissions.js

```diff
@@ -16,3 +16,22 @@ export const assertUserEntityPermissionPermissions = async (
   }
   return true;
 };
+
+export const createUserEntityPermissionDBFilter = async (accessPolicy, models, criteria) => {
```

Comment:
Is this _exactly_ the same as the access request one? Can they use a common helper?

Reply:
I've abstracted this out into a `createAdminPanelDBFilter` utility